packages/flutter/lib/src/rendering/paragraph.dart

```diff
@@ -735,6 +735,11 @@ class RenderParagraph extends RenderBox
     }
   }
 
+  /// An estimate of the height of a line in the text. See [TextPainter.preferredLineHeight].
+  /// This does not required the layout to be updated.
+  double get preferredLineHeight => _textPainter.preferredLineHeight;
```

Comment:
If you have a workaround for preferredLineHeight, I would prefer that we hold off on exposing this as official API. The current implementation of this value is sketchy at best and is something I want to eventually rework.

Some problems with this value is that it assumes an alphabetic space as the character to get metrics from. This causes some issues with ideographic text such as Chinese or even Arabic in some fonts. The current implementation also does not account for the differences in first line vs second+ line height, so could produce values that are not useful beyond the first line.

For now, I suggest continuing to use your workaround instead of adding this API that may have to be broken in the future.

Reply:
Makes sense, thanks! I can keep my workaround for now and remove this method from here.